Formula/pyside.rb

```diff
@@ -17,27 +17,39 @@ class Pyside < Formula
   depends_on "qt"
 
   def install
+    homebrew_args = %W[
+      --no-user-cfg
+      install
+      --prefix=#{prefix}
+      --install-scripts=#{bin}
+      --single-version-externally-managed
+      --record=installed.txt
+    ]
+
     args = %W[
```

Comment:
Can you combine these two arrays into a single array including `--install-lib` and `site-packages`?

Reply:
The destination path is different for `python` and `python@2`. Would you prefer that I define two args variables or reuse the same array name and change the `--install-lib` argument?

Reply:
Done, thanks for reviewing this.